src/index.html

```diff
@@ -1,12 +1,337 @@
+          </div>
+          <button class="button card__button">
+            Buy
+          </button>
+        </div>
+
+        <div class="card section__card">
+          <img
+            src="./images/imac.jpeg"
+            alt="iMac"
+            class="card__img"
+          >
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="card__code">Product code: 195434</p>
+          <div class="card__rating">
+            <ul class="stars stars--4">
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+            </ul>
+            <span class="card__rating-review">Reviews: 5</span>
+          </div>
+          <div class="card__price">
+            <span class="card__price-text">Price:</span>
+            <span class="card__price-value">$2,199</span>
+          </div>
+          <button class="button card__button">
+            Buy
+          </button>
+        </div>
+
+        <div class="card section__card">
+          <img
+            src="./images/imac.jpeg"
+            alt="iMac"
+            class="card__img"
+          >
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="card__code">Product code: 195434</p>
+          <div class="card__rating">
+            <ul class="stars stars--4">
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+            </ul>
+            <span class="card__rating-review">Reviews: 5</span>
+          </div>
+          <div class="card__price">
+            <span class="card__price-text">Price:</span>
+            <span class="card__price-value">$2,199</span>
+          </div>
+          <button class="button card__button">
+            Buy
+          </button>
+        </div>
+
+        <div class="card section__card">
+          <img
+            src="./images/imac.jpeg"
+            alt="iMac"
+            class="card__img"
+          >
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="card__code">Product code: 195434</p>
+          <div class="card__rating">
+            <ul class="stars stars--4">
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+            </ul>
+            <span class="card__rating-review">Reviews: 5</span>
+          </div>
+          <div class="card__price">
```

Comment:
would be good to separate elements (by elements I mean those in understanding by BEM)

Reply:
Sorry I didn't understand what you mean here. All of them card element so I named them under the card element. How should have I done it?